packages/core/src/middleware/index.ts

```diff
@@ -8,4 +8,5 @@ export { Keygrip } from "./interfaces/cookie-session/keygrip.interface";
 export { CookieParserOptions } from "./interfaces/cookie-parser.interface";
 export { ServeFaviconOptions } from "./interfaces/serve-favicon.interface";
 export { RateLimitOptions } from "./interfaces/express-rate-limit.interface";
+export { MulterOptions } from "./interfaces/multer.interface";
```

Comment:
Make sure you return all types related to your options. I think MulterOptions is inside of namespace Multer that's why build process failed.